app/pages/patientdata/patientdata.js

```diff
@@ -76,7 +76,7 @@ var PatientData = React.createClass({
         },
         timePrefs: this.props.timePrefs
       },
-      chartType: 'daily',
+      chartType: 'modal',
```

Comment:
is it worth being pedantic and calling this something like defaultChart: 'modal' ??